### DISCUSSION TITLE
Add login to MOTD

### DISCUSSION THREAD

**Discussion by VoidLock** [community-scripts/ProxmoxVE#533] [Q&A] [answered]
So just an idea, can we add 

> Frigate should be reachable by going to the following URL.
         http://192.168.86.250:5000 

to the MOTD? It is already in the script to add "****** LXC provided by https://helper-scripts.com/", but if we could add the current IP of the machine PLUS the port (if needed) to the MOTD so when you open the console for that LXC it will prompt you with the reachable address that has been installed. 

I say the "Current" IP because after the script finishes, for me at least, I go in and change the IP & VLAN to a separate LXC/Docker VLAN that I have set up, so simply adding the finishing line to MOTD would not help me per say. I'm sure that many other users have the same routine when they finish a script.

**Comment by Mellowlynx** [comment, marked as answer]
#180
